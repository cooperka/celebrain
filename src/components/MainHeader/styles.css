.MainHeader {
  display: flex;
  flex-direction: column;
  justify-content: center;
  min-height: 300px;
  margin: 80px 0 0 0; /* Top margin to account for the nav bar. */
  padding-top: 2em;
  color: #444;
  font-weight: bold;
}

.MainHeader .title {
  margin: 1rem;
  font-size: 5em;
}

.MainHeader .subtitle {
  margin: 1rem auto;
  font-size: 2.2em;
  max-width: 20em;
}

.MainHeader .buttons {
  margin: 1rem auto;
}
