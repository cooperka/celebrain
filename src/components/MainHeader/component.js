// @flow

import React from 'react';

import './styles.css';

function MainHeader() {
  return (
    <div className="MainHeader">
      <div className="title">Lovable. Simple. Refined.</div>
      <div className="subtitle">
        I listen to users and build the <i>minimal&nbsp;lovable&nbsp;solution</i> to their needs.
      </div>
    </div>
  );
}

export default MainHeader;
