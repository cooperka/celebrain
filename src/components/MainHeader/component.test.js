// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import MainHeader from './component';

it('renders', () => {
  const tree = renderer.create(<MainHeader />);
  expect(tree.toJSON()).toMatchSnapshot();
});
