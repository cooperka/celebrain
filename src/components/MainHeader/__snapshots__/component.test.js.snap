// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="MainHeader"
>
  <div
    className="title"
  >
    Lovable. Simple. Refined.
  </div>
  <div
    className="subtitle"
  >
    I listen to users and build the 
    <i>
      minimal lovable solution
    </i>
     to their needs.
  </div>
</div>
`;
